src/cloudsc2_nl_loki/cloudsc_loki.config

```diff
@@ -0,0 +1,51 @@
+[default]
+# Specifies the behaviour of auto-expanded routines
+role = 'kernel'
+expand = true  # Automatically expand subroutine calls
+strict = true  # Throw exceptions during dicovery
+
+demote_locals = true
+
+# Ensure that we are never adding these to the tree, and thus
+# do not attempt to look up the source files for these.
+# TODO: Add type-bound procedure support and adjust scheduler to it
+
+disable = ['performance_timer%start', 'performance_timer%end', 'performance_timer%thread_start',
+           'performance_timer%thread_end', 'performance_timer%thread_log',
+           'performance_timer%thread_log', 'performance_timer%print_performance']
+
+# Define entry point for call-tree transformation
+[[routine]]
+name = 'cloudsc_driver'
+expand = true
+role = 'driver'
+
+[[routine]]
+name = 'cloudsc2'
+role = 'kernel'
+expand = true
+demote_locals = false
```

Comment:
Ohhhhh? :open_mouth: Is locals demotion disabled for a reason here? Does it change/impact results?